src/Session/profile.py

```diff
@@ -1,10 +1,34 @@
 from datetime import date
+from src.Constants.keys import *
+from src.FoodStorage.foodStorage import FoodStorage
+from src.FoodStorage.freezer import Freezer
+from src.FoodStorage.fridge import Fridge
 
 class Profile:
     def __init__(self, name):
         self.name = name
         self.current_date = date.today()
-        self.foodStorages = {}
+        self._foodStorages = {}
+        self._opened_foodStorage = None
+
+    def addFoodStorage(self, foodStorage_type, name):
+        if foodStorage_type not in self._foodStorages:
+            self._foodStorages[foodStorage_type] = {}
+        
+        newFoodStorage = FoodStorage()
+        if foodStorage_type == FREEZER:
+            newFoodStorage = Freezer()
+        elif foodStorage_type == FRIDGE:
+            newFoodStorage = Fridge()
+
+        self._foodStorages[foodStorage_type][name] = newFoodStorage
+        self._opened_foodStorage = self._foodStorages[foodStorage_type][name]
+
+    def getFoodStorage(self, foodStorage_type, name):
+        if foodStorage_type not in self._foodStorages or \
+            name not in self._foodStorages[foodStorage_type]:
+            return None
+        return self._foodStorages[foodStorage_type][name]
```

Comment:
Yes this is completely different from `addFoods`. This is for adding a new `FoodStorage` object and getting existing `FoodStorage` objects. 

Basically the interaction would be like this:
User runs our application
User types in the command `create profile` and proceeds to create a profile OR they load an existing profile
User types in the command `create freezer` 
In the background, what's happening is we will redirect to the `create_freezer()` method in `commands.py` which will walk through asking the user for a name for the freezer AND check if that already exists on this profile. 
When we check if the freezer exists, we're calling `getFoodStorage()` which returns the `FoodStorage` object if it exists in `self._foodStorages`
When are actually add a new Freezer, we call `addFoodStorage()` which will create a new `FoodStorage()` object, depending on the type you're creating (fridge, freezer, etc) and add it under the profiles `self._foodStorages`. 

I added `newFoodStorage` to swap what kind of `FoodStorage` object we were creating based on the type (freezer, fridge, etc) but default to a general `FoodStorage` object.

Reply:
I'm realizing I forgot to add descriptions for the methods though, so I'll go ahead and add that!

Reply:
Also for your question on `addFoods`, we will be doing something similar to what we've done here, except it will be when the user types `add food`. In that case, we'll redirect to `add_food()` in `commands.py` which will eventually call `addFoods` in `FoodStorage`. Basically we're tying everything together! But we need to do it by grabbing the user's commands and redirecting it to a method specifically for that command that will use our code that we wrote earlier for `Food` and `FoodStorage`.

Hopefully that makes sense!

Reply:
Got it! Thanks for explaining all that for me, really appreciate it! It makes much more sense now!